src/components/ActionMenu/components/MenuGroup/MenuGroup.tsx

```diff
@@ -9,14 +9,14 @@ import MenuAction from '../MenuAction';
 import styles from './MenuGroup.scss';
 
 export interface Props extends MenuGroupDescriptor {
-  /** Visually hidden text for screen readers */
+  /** Visually hidden menu description for screen readers */
   accessibilityLabel?: string;
-  /** Show or hide the MenuGroup */
+  /** Whether or not the menu is open */
   active?: boolean;
   /** Callback for opening the MenuGroup by title */
   onOpen(title: string): void;
   /** Callback for closing the MenuGroup by title */
-  onClose(): void;
+  onClose(title: string): void;
```

Comment:
Curious why are we passing the title `onClose`? Was this previous functionality?

Reply:
Yes, this was the function signature before #1653

Reply:
Gotcha, makes sense 😄